config/test.json

```diff
@@ -6,7 +6,8 @@
     "truncateLog": true,
     "metricPrefix": "test",
     "datadog": {
-      "apiKey": "some-api-key"
+      "apiKey": "some-api-key",
+      "batchSize": 1
```

Comment:
Style issue I guess, but for me a batchSize of 1 implies that every message will be sent separately.

I'd prefer to see it set to 2 here and the logic in the check changed to `<= batchSize`

Reply:
After discussion with you I leave this as is since it's intention and I reassign the batchSize to 2 in test. 1 is for being able to send one shots right away.